app/controllers/super_admins/external_users_controller.rb

```diff
@@ -17,6 +17,8 @@ def new
   end
 
   def create
+    # downcase email_confirmation - devise will downcase the email
+    params[:external_user][:user_attributes][:email_confirmation].downcase!
```

Comment:
You're coupling code implementation to configuration. It's not going to be changed during runtime, but can be changed nevertheless.